docs/conf.py

```diff
@@ -197,7 +197,9 @@ def __getattr__(cls, name):
     edit_on_github_doc_root = "docs"
 
 nitpicky = True
-nitpick_ignore = [('py:class', 'astroquery.mast.core.MastQueryWithLogin')]
+nitpick_ignore = [('py:class', 'astroquery.mast.core.MastQueryWithLogin'),
+                  # astropy interited type annotations
```

Comment:
Belated comment: `interited` -> `inherited`

Reply:
It is unfortunate this is needed but I see that Marten already commented upstream. Thanks!